seqr/utils/file_utils.py

```diff
@@ -81,17 +81,19 @@ def mv_file_to_gs(local_path, gs_path, user=None):
     _run_gsutil_with_wait(command, gs_path, user)
 
 
-def get_gs_file_list(gs_path, user=None):
+def get_gs_file_list(gs_path, user=None, check_subfolders=True):
     gs_path = gs_path.rstrip('/')
-    command = 'ls'
+    command = '-q ls'
```

Comment:
why is this changing?

Reply:
Adding the `-q` option to the `gsutil` command disables reporting progress and other information.

Reply:
The `-q` option doesn't change the behaviors of the `ls` command. I will take it out.